src/main/java/com/synopsys/integration/alert/web/config/FieldModelProcessor.java

```diff
@@ -101,16 +101,34 @@ public FieldModel performUpdateAction(final FieldModel fieldModel) {
 
     public Map<String, String> validateFieldModel(final FieldModel fieldModel) {
         final Map<String, String> fieldErrors = new HashMap<>();
-        final Optional<DescriptorActionApi> descriptorActionApi = retrieveDescriptorActionApi(fieldModel);
+        final FieldModel trimmedFieldModel = trimFieldModelValues(fieldModel);
+        final Optional<DescriptorActionApi> descriptorActionApi = retrieveDescriptorActionApi(trimmedFieldModel);
         if (descriptorActionApi.isPresent()) {
-            final Map<String, ConfigField> configFields = retrieveUIConfigFields(fieldModel.getContext(), fieldModel.getDescriptorName())
+            final Map<String, ConfigField> configFields = retrieveUIConfigFields(trimmedFieldModel.getContext(), trimmedFieldModel.getDescriptorName())
                                                               .stream()
                                                               .collect(Collectors.toMap(ConfigField::getKey, Function.identity()));
-            descriptorActionApi.get().validateConfig(configFields, fieldModel, fieldErrors);
+            descriptorActionApi.get().validateConfig(configFields, trimmedFieldModel, fieldErrors);
         }
         return fieldErrors;
     }
 
+    public FieldModel trimFieldModelValues(final FieldModel fieldModel) {
```

Comment:
Why would we only want to do this for validation.  When creating a configuration or updating wouldn't we want to save just the necessary values.  I think we would want this trim method in the save and update methods as well.  But it would need to be tested.